exercises/two-fer/.meta/src/reference/kotlin/Twofer.kt

```diff
@@ -0,0 +1,10 @@
+fun twofer(name: String = ""): String {
+    var tempName = name
+    val regex = Regex("[A-Za-z]")
+    val matched = regex.containsMatchIn(tempName)
+
+    if(!matched)
+        tempName = "you"
+
+    return "One for $tempName, one for me."
```

Comment:
In light of the above, this sample implementation can be reduced to a single line.